translate/cloud-client/quickstart.py

```diff
@@ -24,15 +24,21 @@ def run_quickstart():
     api_key = 'YOUR_API_KEY'
 
     # Instantiates a client
-    translate_client = translate.Client(api_key)
+    translate_client = translate.Client(api_key=api_key)
 
     # The text to translate
     text = u'Hello, world!'
     # The target language
     target = 'ru'
 
+    # MT model type `base` or `nmt`
+    model = 'base'
```

Comment:
This will probably be a constant as per https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2731#discussion_r87871349